### ISSUE TITLE
train.py: error: the following arguments are required: --img-list

### ISSUE THREAD

**Issue by tphankr** [voxelmorph/voxelmorph#355]
Thank you for your code.
Your code is excellent but difficult for me use. In short, I do not know how to use the voxelmorph code yet, although I tried to many run times (> 4 times) . Please help me some questions.
1) I tried to do Training step: 
./scripts/tf/train.py /path/to/training/data --model-dir /path/to/models/output --gpu 0
And, I typed like this but had the error --img-list. How do I fill this request? Thank you.
![image](https://user-images.githubusercontent.com/37217354/135755937-3b50392f-0124-4434-965c-61532a346da3.png)
2) I do not know about the data folder. I mean I do not know, how to load the .npz file in this folder when I train.
For example, which .npz file is Moving image or Fixed image, respectively?
![image](https://user-images.githubusercontent.com/37217354/135755824-29539a50-53a4-43e2-90b8-a093ec59c331.png)

Thank you very much.

**Comment by WwwwJX** [comment]
Excuse me, I meet the same question, so how do you solve it??

**Comment by WwwwJX** [comment]
> Thank you for your code. Your code is excellent but difficult for me use. In short, I do not know how to use the voxelmorph code yet, although I tried to many run times (> 4 times) . Please help me some questions.
> 
> 1. I tried to do Training step:
>    ./scripts/tf/train.py /path/to/training/data --model-dir /path/to/models/output --gpu 0
>    And, I typed like this but had the error --img-list. How do I fill this request? Thank you.
>    ![image](https://user-images.githubusercontent.com/37217354/135755937-3b50392f-0124-4434-965c-61532a346da3.png)
> 2. I do not know about the data folder. I mean I do not know, how to load the .npz file in this folder when I train.
>    For example, which .npz file is Moving image or Fixed image, respectively?
>    ![image](https://user-images.githubusercontent.com/37217354/135755824-29539a50-53a4-43e2-90b8-a093ec59c331.png)
> 
> Thank you very much.

Excuse me, I meet the same question, so how do you solve it??

**Comment by adalca** [comment]
Our apologies I think we changed the default way to feed in data recently.
the IMG_LIST should be a text file, where each line is a file in your dataset.

For example, you can download the brain data from 
https://github.com/adalca/medical-datasets/blob/master/neurite-oasis.md
then your IMG_LIST can be a list of the files you want to train with from that dataset, like:

```
/path/to/neurite/oasis/OASIS_OAS1_0001_MR1/slice_norm.nii.gz
/path/to/neurite/oasis/OASIS_OAS1_0002_MR1/slice_norm.nii.gz
...
```

@ahoopes is this right?

**Comment by ahoopes** [comment]
Yes, this is correct. Sorry for the confusion. I've updated the README with new instructions.

Basically, you'll need some file that lists the unique image filenames in your training dataset. This doesn't necessarily have to include the full paths though, as you can use the `--img-prefix` and `--img-suffix` flags to add information. For example, in the OASIS dataset that Adrian linked, you can use `subjects.txt` (included in the dataset) as your file list, and use `--img-prefix /path/to/downloaded/dataset/` and `--img-suffix /norm.nii.gz` to specify where the data is stored and which image file to use from the subject folders.

From the updated README:

>  For a given image list file `/images/list.txt` and output directory `/models/output`, the following script will train an image-to-image registration network (described in MICCAI 2018 by default) with an unsupervised loss. Model weights will be saved to a path specified by the `--model-dir` flag.
> 
> ```
> ./scripts/tf/train.py --img-list /images/list.txt --model-dir /models/output --gpu 0
> ```
> 
> The `--img-prefix` and `--img-suffix` flags can be used to provide a consistent prefix or suffix to each path specified in the image list.

**Comment by tphankr** [comment]
Dear @adalca and @ahoopes ...
These months, I read your comment again and again with this training step and do but not do yet.  Thank you very much.
**Can you provide comments or suggestions more about my processing with this step to I can run this training step?** I faced the problem about the TensorFlow as following( I aslo tried to do many times to re-install the environment but now I am stucked, so now I post on here) 
![image](https://user-images.githubusercontent.com/37217354/147370405-d30cd57f-a315-4fe4-8625-706d2593d87f.png)

                                ---------my do with your neurite-oasis.v1.0 dataset---------

1/2) make a subjects.txt with 4 folder (actualy your data have 414 folder), as following: 
![image](https://user-images.githubusercontent.com/37217354/147370162-0fc7ffa4-167f-4f3d-bfda-e6f45d738a83.png)
![image](https://user-images.githubusercontent.com/37217354/147370206-a0a658b2-4549-4343-bfaa-738a56f3b436.png)

2/2) And then, I typed ./scripts/tf/train.py --img-list ./Data/neurite-oasis.v1.0/subjects.txt --model-dir ./models/output/ --gpu 1
**It is look like run around the 3 seconds successfully  but HAVE THE PROBLEM OF TensorFlow as above following:**
![image](https://user-images.githubusercontent.com/37217354/147370575-6c63dc5d-8848-4cd4-a6a8-5729e9005fe5.png)
![image](https://user-images.githubusercontent.com/37217354/147370642-9a833ee1-43e3-4cff-9357-e6f5694e23bb.png)
![image](https://user-images.githubusercontent.com/37217354/147370686-de1d6629-8074-4cd3-9748-01392c0fd094.png)

                      ---------------------- the end ---------------------

Thank you very much.

**Comment by ahoopes** [comment]
Those steps look correct to me. I just tested things with a fresh env and did not run into any issues. What tensorflow version are you using? and are you sure that you're using the recent dev version of voxelmorph/neurite?

**Comment by tphankr** [comment]
Thank you @adalca and @ahoopes who hinted some keyworks to can solved. 
I solved the train step by tensorflow 2.2.0.

Thank you very much.

**Comment by domadaaaa** [comment]
> Our apologies I think we changed the default way to feed in data recently. the IMG_LIST should be a text file, where each line is a file in your dataset.
> 
> For example, you can download the brain data from https://github.com/adalca/medical-datasets/blob/master/neurite-oasis.md then your IMG_LIST can be a list of the files you want to train with from that dataset, like:
> 
> ```
> /path/to/neurite/oasis/OASIS_OAS1_0001_MR1/slice_norm.nii.gz
> /path/to/neurite/oasis/OASIS_OAS1_0002_MR1/slice_norm.nii.gz
> ...
> ```
> 
> @ahoopes is this right?

Hi, train_list includes moving_image or fixed_image?

**Comment by domadaaaa** [comment]
> Our apologies I think we changed the default way to feed in data recently. the IMG_LIST should be a text file, where each line is a file in your dataset.
> 
> For example, you can download the brain data from https://github.com/adalca/medical-datasets/blob/master/neurite-oasis.md then your IMG_LIST can be a list of the files you want to train with from that dataset, like:
> 
> ```
> /path/to/neurite/oasis/OASIS_OAS1_0001_MR1/slice_norm.nii.gz
> /path/to/neurite/oasis/OASIS_OAS1_0002_MR1/slice_norm.nii.gz
> ...
> ```
> 
> @ahoopes is this right?

hi, your 

> Our apologies I think we changed the default way to feed in data recently. the IMG_LIST should be a text file, where each line is a file in your dataset.
> 
> For example, you can download the brain data from https://github.com/adalca/medical-datasets/blob/master/neurite-oasis.md then your IMG_LIST can be a list of the files you want to train with from that dataset, like:
> 
> ```
> /path/to/neurite/oasis/OASIS_OAS1_0001_MR1/slice_norm.nii.gz
> /path/to/neurite/oasis/OASIS_OAS1_0002_MR1/slice_norm.nii.gz
> ...
> ```
> 
> @ahoopes is this right?

hi, you use slice_norm,nii in example txt file, it means that 2D registration?
